### DISCUSSION TITLE
Markdown

### DISCUSSION THREAD

**Discussion by isidroas** [pycontribs/ansi2html#209] [Q&A]
Does the html output of this application is supported by Gitlab/Github markdown ?

**Comment by hartwork** [comment]
I don't get the question. Please elaborate or rephrase.

**Reply by isidroas** [reply]
I meant to put colors in Github Flavored markdown. 
But I realized that is not possible:  https://stackoverflow.com/a/24174358/16926605

**Reply by hartwork** [reply]
ansi2html does not support markdown at all. I'll close this discussion then.
